### PR TITLE
1/3: Remove post-install hook

### PR DESCRIPTION
**Pull request by hgeraldino**
First PR: remove post-install hook

### CODE REVIEW

File: src/test/resources/features/checksum_verification.feature [hunk 1]
```diff
@@ -71,11 +71,11 @@ Feature: Verify checksums
...
 
-	Scenario: Abort installation after download of a binary with invalid MD5 checksum
-		Given the system is bootstrapped
-		And the candidate "grails" version "1.3.9" is available for download with checksum "1e87a7d982a2f41da96fdec289908533" using algorithm "MD5"
-		When I enter "sdk install grails 1.3.9"
-		Then I see "Stop! An invalid checksum was detected and the archive removed! Please try re-installing."
-		And the candidate "grails" version "1.3.9" is not installed
-		And the archive for candidate "grails" version "1.3.9" is removed
-		And the exit code is 1
+#	Scenario: Abort installation after download of a binary with invalid MD5 checksum
```
Comment (marc0der) [inline src/test/resources/features/checksum_verification.feature:74]: Curious about why this was commented out. We're still calling the `__sdkman_checksum` function above in `sdkman-install.sh`. Was this perhaps causing some failure?

File: src/test/resources/features/pre_and_post_installation_hooks.feature [hunk 2]
```diff
@@ -11,10 +11,10 @@ Feature: Hooks
 		And the system is bootstrapped
 		And the candidate "grails" version "2.1.0" is available for download on "Linux" with architecture "x86_64"
 		And a "pre" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns successfully
-		And a "post" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns successfully
+#		And a "post" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns successfully
```
Comment (marc0der) [inline src/test/resources/features/pre_and_post_installation_hooks.feature:14]: Is it worth stripping this out of the step defs if we're not longer going to us it?

File: src/test/resources/features/pre_and_post_installation_hooks.feature [hunk 3]
```diff
@@ -26,12 +26,12 @@ Feature: Hooks
...
-	Scenario: Post-install Hook returns a non-zero code
-		And an "x86_64" machine with "Linux" installed
-		And the system is bootstrapped
-		And the candidate "grails" version "2.1.0" is available for download on "Linux" with architecture "x86_64"
-		And a "pre" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns successfully
-		And a "post" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns a failure
-		When I enter "sdk install grails 2.1.0"
-		Then I see "Post-installation hook failure"
-		And the exit code is 1
+#	Scenario: Post-install Hook returns a non-zero code
```
Comment (marc0der) [inline src/test/resources/features/pre_and_post_installation_hooks.feature:29]: I guess we can delete this one completely.

Comment (marc0der) [reply src/test/resources/features/pre_and_post_installation_hooks.feature:29]: On second thought, I think this entire feature file and all it's related step defs can go. I also suggest ripping out the pre-install hooks as part of this. We no longer use those either.

**Review by marc0der — CHANGES_REQUESTED** [review]
Looking good! I think we can rip out the nasty pre-installation hooks too, since we don't use them anymore.

File: src/test/resources/features/checksum_verification.feature (hunk 1 above)
Comment (marc0der) [reply src/test/resources/features/checksum_verification.feature:74]: Don't worry, just saw the follow up PR :smile:

File: src/test/resources/features/java_installation.feature [hunk 4]
```diff
@@ -11,34 +11,34 @@ Feature: Java Multi Platform Binary Distribution
...
 
-	Scenario: Platform is supported and a specific version of compatible binary is installed
-		Given an "x86_64" machine with "Linux" installed
-		And the system is bootstrapped
-		And the candidate "java" version "8.0.111" is available for download on "Linux" with architecture "x86_64"
-		And the appropriate multi-platform hooks are available for "java" version "8.0.111" on "Linux" with architecture "x86_64"
-		When I enter "sdk install java 8.0.111"
-		And I see "Done installing!"
-		And the candidate "java" version "8.0.111" is installed
+#	Scenario: Platform is supported and a specific version of compatible binary is installed
```
[the remark this answers is not in the document]
Comment (marc0der) [reply src/test/resources/features/java_installation.feature:14]: Same as before, please ignore this.

**Comment by hgeraldino** [comment]
> Looking good! I think we can rip out the nasty pre-installation hooks too, since we don't use them anymore.

Removed

**Review by marc0der — APPROVED** [review]

**Pull request merged**